motif-docs/pages/guides/experimental/maintenance-mode.mdx

```diff
@@ -0,0 +1,46 @@
+---
+heading: Experimental
+title: Maintenance Mode
+subtitle: Temporarily disable traffic to your service using our Maintenance Mode.
+---
+
+import { Image } from '@components/ui/image.mdx'
+
+## About
+
+Supported services: `fargate`, `static`.
+
+The Maintenance Mode allows you to temporarily divert traffic from your service to a page that informs your users that the service is temporarily unavailable.
+The page your users will see looks like this:
+
+<div className="drop-shadow-md rounded min-h-[200px]">
+
+<Image
+  className="margin-0 rounded"
+  height="200"
+  src="https://res.cloudinary.com/djp21wtxm/image/upload/v1666430995/i1600x983-LPvqk9j7nJVD_exnscl.png"
+/>
+</div>
+
+## How to enable it
```

Comment:
```suggestion
## How To Enable It
```

Reply:
@flybayer Just to share, I personally dislike capitalising every word on a title :D 
Also, I'm pretty sure it's considered "incorrect" to capitalise words like `to` and other prepositions/small words according to a few style guides.